### DISCUSSION TITLE
Is there an archive of (NASA) (mission control) dashboards for inspiration?

### DISCUSSION THREAD

**Discussion by abitrolly** [nasa/openmct#7269] [Community Q&A]
Does OpenMCT use Grafana for its dashboards? I am quite disappointed that all the beautiful stuff we've seen in SciFi movies, anime and plain human made art came down to this.

In my area of DevOps, designers are a scarce human resource. Even if you manage to get some of these beautiful creatures with white marble tablets sitting their laps, producing unimaginable shininess under their pens - even if you get one for your team - the first bypassing manager will not resist to steal the beauty from you, to draw own presentations, web-sites and visit cards. That's why we are doomed to stare endlessly at the grayness of functional and dull Grafana plots with no hope of beauty. The space missions is our only chance to escape that gravity.

Technologies not only gives us the tools, they also limit our imagination. We are quick to snap to solutions we've seen rather than giving somebody a chance to reinvent something new, make mistakes, and pass the knowledge.

And making a dashboard is not an easy task - it needs knowledge of signals, of how signals are passed, of how they are collected, which information is extracted from them, which information is made visible, and which is actionable. Now lets' add "visually appealing" and "inspiring" to that mix.

Is there an archive of mission control dashboards from the past with the description of signals displayed, so that a new generation of designers (and not DevOps like me) could propose visual alternatives to what we have now?

It would be also awesome to see a collection of artistic dashboards, so that somebody can thoroughly review what is missing form the functional side for them to be used for a real project.

**Comment by tristan-f-r** [comment]
[OpenMCT does have a visual designer](https://github.com/nasa/openmct/issues/created_by/rukmini-bose), by the way.

**Reply by abitrolly** [reply]
Thanks for the pointer - it is nice to get see some people with experience here. Although I would argue that UX Designer is not a visual designer. I bet @rukmini-bose can explain better. )

Have nice holidays!

**Reply by tristan-f-r** [reply]
Oh, whoops! Yeah, I didn't notice that she was a UX designer.
